libres/lib/enkf/misfit_ensemble.cpp

```diff
@@ -184,7 +169,12 @@ void misfit_ensemble_fread(misfit_ensemble_type *misfit_ensemble,
 }
 
 misfit_ensemble_type *misfit_ensemble_alloc() {
-    misfit_ensemble_type *table = misfit_ensemble_alloc_empty();
+    misfit_ensemble_type *table =
```

Comment:
When you are here I would suggest doing `misfit_ensemble_type * table = new misfit_ensemble_type();` and `delete table` in the `misfit_ensemble_free()` function.